importer/Bosnia.py

```diff
@@ -98,20 +97,25 @@ def set_is(self):
 
         self.add_statement("is", to_add, refs=self.source, if_empty=True)
 
-    def set_described(self):
+    def clean_url(self):
         """
-        Set described at URL.
+        Prettify reference url.
 
         The URL's look like
         http://aplikacija.kons.gov.ba/kons/public/nacionalnispomenici/show/3729?return=&search=idgrad:undefined:46|
         which is messy so the mess after ? gets removed. Same result!
         """
         if self.has_non_empty_attribute("url"):
             if self.url.startswith("http"):
-                url_clean = self.url.split("?")[0]
-                self.add_statement("described_at_url",
-                                   url_clean,
-                                   refs=self.source)
+                return self.url.split("?")[0]
+
+    def set_described(self):
```

Comment:
Do we actually want this as a qualifier on heritage status?

Reply:
This is not being added as a qualifier on heritage status.

Reply:
oh god now i get it